idd/Energy+.idd.in

```diff
@@ -85417,8 +85417,7 @@ ExternalInterface:FunctionalMockupUnitExport:To:Schedule,
        \retaincase
   N1 ; \field Initial Value
        \type real
-       \required-field
-       \note Used during the first call of EnergyPlus.
+       \note Used during EnergyPlus sizing and warm-up.
```

Comment:
NumErrorFlag is set. But that's not used later to throw.

Is allowing a blank really intended though, given the warning?

https://github.com/NREL/EnergyPlus/blob/0befe779fe9721745ba31a65f877cc05905ac6b3/src/EnergyPlus/ScheduleManager.cc#L2047-L2053

Reply:
For the record, looking further at this code, it appears that this is creating a new schedule, so it would need a value to initialize it.